p2p/discovery.py

```diff
@@ -412,6 +412,7 @@ def datagram_received(self, data: Union[bytes, Text], addr: Tuple[str, int]) ->
         address = kademlia.Address(ip_address, udp_port)
         # The prefix below is what geth uses to identify discv5 msgs.
         # https://github.com/ethereum/go-ethereum/blob/c4712bf96bc1bae4a5ad4600e9719e4a74bde7d5/p2p/discv5/udp.go#L149  # noqa: E501
+        data = snappy.decompress(data)
```

Comment:
We prefer not to overwrite variables.  I'd adjust the "compressed" value to be `raw_data` and the decompressed value to be assigned to `data`.

Reply:
@pipermerriam will do the changes as requested. Also I think tox.ini has to be modified so that it sets up the environment accordingly. Any insights into how I can do the tox.ini modification?

Reply:
The modification likely needs to take place in the `./.circleci/config` file to do the `apt` install.

You can see how that is done in the `py-snappy` repo here: https://github.com/ethereum/py-snappy/blob/master/.circleci/config.yml#L23-L25

Reply:
@pipermerriam I believe that some modifications need to be done to the tox.ini file too, because a lot of them are failing because of tox